Resources/Prototypes/_Backmen/Research/experimental.yml

```
@@ -44,3 +44,27 @@
+
+- type: technology
+  id: NightVisionTech
+  name: research-technology-night-vision
+  icon:
+    sprite: Backmen/Clothing/Eyes/Goggles/nightvision.rsi
+    state: icon
+  discipline: Experimental
+  tier: 2
+  cost: 10000
+  recipeUnlocks:
+  - ClothingEyesNightVisionGoggles
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)